.gitattributes

```diff
@@ -1,9 +1,11 @@
-/doc export-ignore
-/test export-ignore
-/vendor export-ignore
-.gitattributes export-ignore
-.gitignore export-ignore
-.scrutinizer.yml export-ignore
-.travis.yml export-ignore
-phpunit.xml.dist export-ignore
-phpcs.xml export-ignore
+/.coveralls.yml export-ignore
```

Comment:
Could you please re-do this patch without changing the sorting of entries? This diff is really noisy, plus `git blame` information is lost.

Reply:
Can do. I put it in the same order as the linked projects have.
Would you also like it without the `/` prefix?

Reply:
Simply leave the ones that weren't changed as they are. If the `/` is not present in existing ones, we should also not add it here